pyproject.toml

```diff
@@ -28,15 +28,15 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = ">=3.8,<3.11"
-nibabel = "^2.3.3"
+nibabel = "^3.2.2"
```

Comment:
Could we aim for the latest version (`4.0.2`)? Or is there a limiting factor somewhere?

Reply:
Looks like some of the Nipype's interfaces we depend on need `trackviz` which has been removed in Nibble `4.0.0`...

Reply:
See https://github.com/nipy/nipype/issues/3486

Reply:
Are we actually using the interface Chris mentions in the issue? From the conversation, it looks like it's a very niche use case and the problematic import has been encapsulated deeper since version `1.8.3`.